### PR TITLE
[MLU] fix uts.

### PR DESCRIPTION
**Pull request by ShawnNew**
fix the following uts:
* accuracy
* reduce_max
* reduce_min
* reduce_mean
* reduce_sum
* elementwise_max
* gather_nd
* mean
* sigmoid_cross_entropy_with_logits
* size
* squared_l2_norm
* momentum
* log

### CODE REVIEW

**Review by cifar10 — APPROVED** [review]

**Comment by cifar10** [comment]
LGTM
